### DISCUSSION TITLE
AArch64 binaries for Apple Silicon?

### DISCUSSION THREAD

**Discussion by heliommsfilho** [microsoft/openjdk#7] [Q&A] [answered]
Congratulations, it's a great initiative.

There will be binaries for Apple Silicon?

**Comment by brunoborges** [comment, marked as answer]
Yes, we plan to provide binaries for macOS Apple Silicon in the near future.

**Reply by brunoborges** [reply]
Meanwhile, we have an Early Access binary for Java 16 from November last year.

https://github.com/microsoft/openjdk-aarch64/releases/tag/16-ea%2B10-macos

**Reply by Redth** [reply]
@brunoborges but OpenJdk **11** is still planned for macOS Apple Silicon?

**Reply by karianna** [reply]
We're working on this backport (no promises just yet, but we're pretty motivated :-))

**Reply by nikanorov** [reply]
@karianna are plans for backport 11 still current?

**Reply by karianna** [reply]
Yes - the windows ARM back port has been completed and released as GA by us and is now also in upstream (you'll likely see it in the Jan PSU release for all other vendors).

The M1 back port will follow after that, we're working with Azul and other partners on this. No exact ETA yet but we do have an external EA build that's testing well.
